exercises/practice/sum-of-multiples/tests/sum_of_multiples.cairo

```diff
@@ -0,0 +1,176 @@
+use sum_of_multiples::{sum_of_multiples as verify_sum_of_multiples};
```

Comment:
The aliasing might be confusing to students since the function name in the stub doesn't match the function name used inside the tests. I'd just import `sum_of_multiples` as-is.

Reply:
I had a problem importing `sum_of_multiples`  as the compiler is throwing error of not seeing it as a function but rather as a module because the package is also named `sum_of_multiples`.(this error only happens remotely when the job is being run) that is why I tried to improvise. I checked [sublist](https://github.com/exercism/cairo/blob/eebaaf79e8f35f982db95600ee4b03c95e40d2fb/exercises/practice/sublist/tests/sublist.cairo#L1) and I found it was renamed when imported in test file, so I did same.
another solution is to rename the function.

Reply:
Hmm, if there’s a precedent for it, I suppose you should keep it that way then. Generally though, I’d rename the function in the stub.

Reply:
There's is a precedence for this, but in this case let's rename the function in the stub to `sum` to match the [canonical data](https://github.com/exercism/problem-specifications/blob/main/exercises/sum-of-multiples/canonical-data.json) (see `property` field which indicated expected function name). We then avoid this issue altogether.

Reply:
Noted